src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -224,45 +229,146 @@ def _get_score_modifiers(marqo_query: MarqoQuery) -> \
                 else:
                     raise InternalError(f'Unknown score modifier type {modifier.type}')
 
-            # Note one of these could be empty, but not both
-            return {
-                unstructured_common.QUERY_INPUT_SCORE_MODIFIERS_MULT_WEIGHTS: mult_tensor,
-                unstructured_common.QUERY_INPUT_SCORE_MODIFIERS_ADD_WEIGHTS: add_tensor
-            }
+            if self._marqo_index_version < self._VERSION_2_10_0:
```

Comment:
What about unstructured index created with version 2.10.x? the new query inputs does not exists yet.

Reply:
I agree. This version checks in this class don't seem right. Should be < 2.11

Reply:
fixed